docker_db.sh

```diff
@@ -177,7 +177,8 @@ EOF
 
 mssql() {
     $CONTAINER_CLI rm -f mssql || true
-    $CONTAINER_CLI run --name mssql -d -p 1433:1433 -e "SA_PASSWORD=Hibernate_orm_test" -e ACCEPT_EULA=Y mcr.microsoft.com/mssql/server:2017-CU13
+    #This sha256 matches a specific tag of mcr.microsoft.com/mssql/server:2019-latest :
+    $CONTAINER_CLI run --name mssql -d -p 1433:1433 -e "SA_PASSWORD=Hibernate_orm_test" -e ACCEPT_EULA=Y mcr.microsoft.com/mssql/server@sha256:f54a84b8a802afdfa91a954e8ddfcec9973447ce8efec519adf593b54d49bedf
```

Comment:
Btw, you might also want to change the version of the image in `Jenkinsfile` on line 164. The duplication is done for security reasons.